ldapauthenticator/ldapauthenticator.py

```diff
@@ -161,7 +249,7 @@ def getConnection(userdn, username, password):
                     username=username,
                     userdn=userdn
             ))
-            conn = ldap3.Connection(server, user=userdn, password=password)
+            conn = ldap3.Connection(server, user=escape_filter_chars(userdn), password=password)
```

Comment:
Actually I had problem with ldap3.Connection() for non-escaped '(' and ')'. At least for Active Directory - I don't know how it would work for other implementations. 
In my case ldap3.Connection().bind() failed with no error message when user parameter had '('  or ')' in it.